transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransport.kt

```diff
@@ -1,32 +1,33 @@
 package com.genesys.cloud.messenger.transport.core
 
-import com.genesys.cloud.messenger.transport.network.DEFAULT_PING_INTERVAL_IN_SECONDS
+import com.genesys.cloud.messenger.transport.network.*
```

Comment:
Might be worth running `./gradlew lintKotlin`. I think it'll complain about the wildcard import here.

Reply:
/me sighs

Reply:
Weird, lint didn't say anything.